src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,164 @@
+
+Transaction events are sampled, so the percentiles we present within Sentry only represents the data we have. Because of sampling, data ranges, filters, or low volume of transactions, we will often have a case where data is directionally correct, but not accurate. For example, the average of one number is that number, but that’s not usually what you may expect to see.
+
+This inability to be usefully accurate will happen more often in some columns than others. We can calculate an average with less data than a 95th percentile, but it’ll also vary by row. An index page will always get more traffic than `/settings/country/belgium/tax`.
+
+### Percentage Query
+
+This column shows the percentage of the entire query that this row represents, so you filter your results with the query and then see how much of the result set a particular transaction represents.
+
+### Requests Per Minute (RPM)
+
+Requests Per Minute is a way to measure throughput. It is the average of all requests bucketed by the minute for the current time window and query string.
+
+## Span
+
+The **span** is the primary building block of a distributed trace, representing an individual unit of work done in a distributed system. It may represent the processing of an HTTP request by a server, the querying of a database, or an HTTP call to an external service.
+
+When you initialize and load a Sentry SDK that supports tracing, it will automatically instrument parts of your running application, such as function calls and network operations. Instrumenting a database call could entail these three steps or subroutines:
+
+1. Calling a function to compose a database query
+2. Sending a query to a database
+3. Receiving and transforming results
+
+Each of these subroutines takes some time to perform (no matter how fast they could run). Sentry records these subroutines as a unit of work in the form of **spans**. Sentry can also attach useful information to spans, such as tags, additional contextual data, and a status to indicate if the subroutine failed or not.
+
+Spans can have descendant spans (or child spans). In our earlier example, the three subroutines can be spans within a larger encompassing span (the database call).
+
+Any instrumented span is part of a trace (identified by its trace id, `trace_id`), and each span has its own marker called the `span_id`, a fixed length of alphanumeric characters.
+
+An example of a span that describes a call to an external service:
+
+```json
+    {
+      "trace_id": "a0fa8803753e40fd8124b21eeb2986b5",
+      "parent_span_id": "9c2a6db8c79068a2",
+      "span_id": "8c931f4740435fb8",
+      "start_timestamp": 1563890702.134,
+      "same_process_as_parent": true,
+      "description": "http://httpbin.org/base64/aGVsbG8gd29ybGQK GET",
+      "tags": { "http.status_code": 200, "status": "ok" },
+      "timestamp": "1563890702.934",
```

Comment:
@untitaker should I change this timestamp?

Reply:
If so, what should I change it to?

Reply:
Yeah please just drop the quotes around the number like for `start_timestamp`